CHANGELOG.md

```diff
@@ -1,5 +1,36 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.29.0 Release
+- Release Date: March 16, 2020
+- Server Versions Supported: Server v5.9+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+ 
+**Note:** An issue was fixed where a user's status was set as online when replying to a message from a push notification. This fix only works in combination with server v5.20.0+.
+ 
+### Improvements
+ - Improved the experience where the app was unresponsive for a period of time after clearing cache or on initial login.
```

Comment:
This doesn't really tell the story of "performance enhancements". Please see the marketing blog for text we can use here